src/docs/markdown/caddyfile/directives/header.md

```diff
@@ -37,28 +37,43 @@ header [<matcher>] [[+|-|?|>]<field> [<value>|<find>] [<replace>]] {
 	?<field> <value>
 
 	[defer]
+
+	# a match condition
+	match [status <code...> | header <field> <value>]
+
+	# or a match block
+	match {
+		status <code...>
+		header <field> <value>
+	}
 }
 ```
 
 - **&lt;field&gt;** is the name of the header field.
 
   With no prefix, the field is set (overwritten).
 
-  Prefix with `+` to add the field instead of overwriting (setting) the field if it already exists; header fields can appear more than once in a request.
+  Prefix with `+` to add the field instead of overwriting (setting) the field if it already exists; header fields can appear more than once in a response.
 
   Prefix with `-` to delete the field. The field may use prefix or suffix `*` wildcards to delete all matching fields.
 
   Prefix with `?` to set a default value for the field. The field is only written if it doesn't yet exist.
 
-  Prefix with `>` to set the field, and enable `defer`, as a shortcut.
+  Prefix with `>` to replace the contents of an existing header value using a regular expression. Implies `defer`, see below.
```

Comment:
Huh? That's not true. `>` is literally only to enable `defer`.

Reply:
Thanks, I had the wrong idea here.